pinot-core/src/main/java/org/apache/pinot/core/util/OsCheck.java

```diff
@@ -46,9 +48,11 @@ public enum OSType {
     Windows, MacOS, Linux, Other
   }
 
+  private static final Logger log = LoggerFactory.getLogger(OsCheck.class);
+
   static {
     String os = System.getProperty("os.name", "generic").toLowerCase(Locale.ENGLISH);
-    System.out.println(os);
+    log.info("System property \"os.name\" is [{}]", os);
```

Comment:
@jasperjiaguo Is this print intentional?

(minor, convention)
```suggestion
    log.info("System property \"os.name\" is: {}", os);
```